lib/index.ts

```diff
@@ -115,7 +116,7 @@ async function extractTargetFrameworksFromProjectConfig(
   manifestFileContents: string): Promise<string[]> {
   try {
     const manifestFile: any = await parseManifestFile(manifestFileContents);
-    return getTargetFrameworksFromProjectConfig(manifestFile);
+    return [...getTargetFrameworksFromProjectConfig(manifestFile)];
```

Comment:
Why are we need to copy array here?

Reply:
I see, it's because of `Set`.

- Let's move Set to array conversion to `getTargetFrameworksFromProjectConfig` (here it's not obvious)
- Are you sure you can use `Set` in node 4?

Reply:
@Kirill89 I can't and I want to deprecate Node 4 😈

Reply:
But we use parser in the CLI, right?